addons/website_sale/static/src/js/website_sale.js

```diff
@@ -638,6 +639,13 @@ export const WebsiteSale = publicWidget.Widget.extend(VariantMixin, cartHandlerM
         }
         return this._changeCountry();
     },
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onChangeState: function (ev) {
+        return Promise.resolve();
```

Comment:
Why is this function returning a Promise ? it works also fine without it :)

Reply:
Yes, it works without it, but I returned a promise for the sake of consistency, as inheritances in both PE and co also return promises.

Reply:
mmmh, I'm not against but I don't really see the point if the function is not async :)